administrator/components/com_content/tables/share.php

```diff
@@ -0,0 +1,30 @@
+<?php
```

Comment:
@nikitadhiman The code is indented too far, it should start at the beginning of the line.